webpack.config.js

```diff
@@ -20,21 +20,37 @@ module.exports = {
     filename: 'bundle.js',
   },
   module: {
-    rules: [{
-      test: /\.(js|jsx)$/,
-      exclude: /node_modules/,
-      use: {
-        loader: 'babel-loader',
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader'],
+      },
+      // global css
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+        exclude: /\.module\.css$/,
+      },
```

Comment:
need the css things anymore?

Reply:
or maybe you just moved them

Reply:
Removed the css stuff from webpack

Reply:
Still need the css rules for `import '@anvilco/react-signature-modal/dist/styles.css'`